server/lib/person.js

```diff
@@ -1,39 +1,33 @@
 const databasePerson = require('../services/person')
 
 class Person {
-  static object (data) {
-    return {
-      personId: data.personid,
-      staffId: data.staffid,
-      firstName: data.firstname,
-      surname: data.surname,
-      email: data.email,
-      gradeId: data.gradeid,
-      locationId: data.locationid,
-      managerId: data.managerid,
-      role: data.role,
-      statusId: data.statusid,
-      skills: [],
-      archived: data.archived
-    }
+  constructor(data) {
+    this.personId = data.personid
+    this.staffId = data.staffid
+    this.firstName = data.firstname
+    this.surname = data.surname
+    this.email = data.email
+    this.gradeId = data.gradeid
+    this.locationId = data.locationid
+    this.managerId = data.managerid
+    this.role = data.role
+    this.statusId = data.statusid
+    this.skills = []
+    this.archived = data.archived
```

Comment:
the static object function is pretty much behaving as a constructor, so you could do it as above, which should also improve the intellisense in VS Code